sdk/identity/Azure.Identity/samples/OtherCredentialSamples.md

```diff
@@ -1,15 +1,19 @@
 # AzurePipelinesCredential Example
 
 This example demonstrates authenticating the `SecretClient` using the `AzurePipelinesCredential` in an Azure Pipelines environment with [service connections](https://learn.microsoft.com/azure/devops/pipelines/library/service-endpoints).
+***Note:*** This credential requires environment variables which are only available in the context of an Azure [PowerShell](https://learn.microsoft.com/azure/devops/pipelines/tasks/reference/powershell-v2?view=azure-pipelines) or [Azure CLI](https://learn.microsoft.com/azure/devops/pipelines/tasks/reference/azure-cli-v2?view=azure-pipelines) task.
```

Comment:
I thought it was the Azure PowerShell task: https://learn.microsoft.com/azure/devops/pipelines/tasks/reference/azure-powershell-v5?view=azure-pipelines. Can you confirm with @KarishmaGhiya?

It would be good to also include this task limitation note in the XML docs for the class or ctor. I assume most customers won't read this doc.

Reply:
Yes it's AzurePowerShell@5 and AzureCLI@2

Reply:
Do we know if those are the minimum required task versions?

Reply:
I'm planning to change this now that we will have the optional params in the ctor.